src/Files.App/Dialogs/ErrorModal.xaml

```diff
@@ -0,0 +1,63 @@
+<ContentDialog
```

Comment:
Formatting check has failed. You may fix it easily by installing and running the xaml styler extension on Visual Studio.

Reply:
Instead of creating a xaml file, I think this modal can be created from code behind using the dialog service.